### PR TITLE
Update review email templates (fixes #1316)

### LINKED ISSUE #1316: Update review email templates
We're changing these templates and some of the canned responses to make them friendlier for developers, and also to update some information in them (in particular, the use of the term 'editors' and the review policy URL).

PR coming.

### PR DESCRIPTION
**Pull request by jvillalobos**
Moved a couple of lines to the base template for consistency, removed
references to editors, and tried to soften the language in a couple of
places. Also updated the review policy URL.

Second try after having lots of fun times with git.

### CODE REVIEW

**Comment by wagnerand** [comment]
r+

File: apps/editors/templates/editors/emails/author_super_review.ltxt [hunk 1]
```diff
@@ -1,5 +1,10 @@
-Your add-on, {{ name }} {{ number }}, has been flagged for Admin Review by an editor.
+Hello,
+
+Your add-on, {{ name }} {{ number }}, has been flagged for Admin Review.
 
 Your add-on is still in our review queue, but it will need to be checked by one of our admin reviewers. This means that your review will probably take longer than usual.
 
-If you have any questions or comments, please reply to this email or join #amo-editors on irc.mozilla.org.
+If you have any questions or comments, you can reply to this email or join #amo-editors on irc.mozilla.org. To learn more about the review process, please visit https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Reviews
```
Comment (muffinresearch) [inline apps/editors/templates/editors/emails/author_super_review.ltxt]: You could remove the locale from the link url so that redirection happens.

Comment (wagnerand) [inline apps/editors/templates/editors/emails/author_super_review.ltxt]: This is not a good idea. The Policies page does not exist in every locale. Non en-US users might get redirected to a non-existent page.

Comment (jvillalobos) [inline apps/editors/templates/editors/emails/author_super_review.ltxt]: Plus, the policies are unlikely to be translated and stay up to date in other locales.

Comment (muffinresearch) [inline apps/editors/templates/editors/emails/author_super_review.ltxt]: Ok, fair enough.

**Pull request merged**